learn/wavemaker-release-notes/v10-6-0.md

```diff
@@ -0,0 +1,117 @@
+
+### Page Cache
+
+- A new flag called **Cache** is introduced on the Page tag. Using this flag, you can specify whether to use a cached page on re-visit. Using a cached page significantly improves render performance. By default, this flag is turned off. To know more about it, see [Cache Flag](/learn/app-development/ui-design/page-concepts/page-cache).
+
+### Session Persistence Horizontal Scaling
+
+- When there is a huge demand for a service, you should plan to scale the application to handle a large number of requests to direct all the requests originating from a single logical client to a single backend web server. Backend servers that use caching to improve performance, or to enable log-in sessions that can benefit from it. To learn more, see [Session Persistence](/learn/app-development/app-security/session-persistence).
+
+
+### Retain State
+
+- To efficiently save and restore UI state by landing on the page at a particular state when sharing the URL, refreshing the page, or by clicking the back button. To learn more, see [Retain UI State in WaveMaker Apps](/learn/blog/2020/11/09/Retain-UI-State-on-Wavemaker-Apps).
+
+### Tablet View
+
+- Supports Developing tablet views in Mobile projects. Access to the tab screens from the Studio. 
+- A new property called **screenType** is introduced which specifies whether the app is rendered on a mobile or tablet. To learn more, see [Designing tablet views in Mobile Project](learn/how-tos/support-for-tablet-view-in-mobile).
+
+## Improvements
+
+---
+
+- You can include the version number in the filename of the exported Cordova zip.
+- Included an `itemsperrow` property for the Checkboxset and Radioset widgets.
+- Required a Warning message for the filter option in the Data Table if bound to query or procedure variable "This will filter the data in each page of the Data Table widget instead of complete data table data set".
+
+### Mobile
+
+- Upgraded to Cordova@10.0.0 and Cordova-ios@6.1.0. [Click here for more details](/learn/blog/2020/11/26/cordova-10.0.0-upgrade).
+- Added the purpose of each iOS icon in the **Export Cordova zip** dialog.
+- Till now, apps created in WaveMaker support both iPhone and iPad by default. There was no control to specify this preference to support only iPhone. In this release, a new preference called 'Target Device' is added under iOS preferences in the **Export Cordova zip** dialog. Using this preference, you can specify whether app supports iPhone or both iPhone and iPad. For old projects, this option is set as 'iPhone and iPad'. For new projects, this option is set as 'iPhone'.
+
+## Bug Fixes
+
+---
+
+- Fixed issue related to allowing Trailing Zeros property in Number widget in a Form and Liveform.
+- Fixed issues in Liveform and Currency widgets related to update value on the **Keypress** property.
+- Allow mapping userAttributes from /security/info api call in the `loggedInUser` variable.
```

Comment:
This should go as an improvement:
"Allow mapping userAttributes from /security/info api call in the `loggedInUser` variable."